api/lib/infrastructure/datasources/airtable/area-datasource.js

```diff
@@ -20,7 +21,8 @@ module.exports = datasource.extend({
       id: airtableRecord.get('id persistant'),
       code: airtableRecord.get('Code'),
       name: airtableRecord.get('Nom'),
-      title: airtableRecord.get('Titre'),
+      titleFrFr: airtableRecord.get('Titre fr-fr'),
```

Comment:
pour le temps de la mise en production et ne pas impacter les utilisateurs actuels, je propose :
```suggestion
      titleFrFr: airtableRecord.get('Titre fr-fr') || airtableRecord.get('Titre'),
```